app/assets/javascripts/active_admin/qiniu_input/upload_image.js

```diff
@@ -37,8 +48,12 @@ $(document).ready(function() {
 
   function uploadImageToQiniu(token, event, callbackFunc) {
     var formData = new FormData();
+    var f = event.target.files[0];
     formData.append('token', token);
-    formData.append('file', event.target.files[0]);
+    formData.append('file', f);
+    if (f.name && f.name.length > 0) {
+      formData.append('key', '' + (new Date()).getTime() + '-' + Math.floor(Math.random() * Math.floor(1000)) + '-' + f.name);
```

Comment:
use a variable for `(new Date()).getTime() + '-' + Math.floor(Math.random() * Math.floor(1000))`